### DISCUSSION TITLE
alerts.atom endpoints

### DISCUSSION THREAD

**Discussion by JJKraw** [weather-gov/api#685] [Q&A] [answered]
According to Service Change Notice 23-121 (https://www.weather.gov/media/notification/pdf_2023_24/scn23-121_alerts_2.3.0.pdf):

> The Atomic Syndication Format (ATOM) formatted data currently served at
alerts.weather.gov will be replaced by the ATOM formatted data available
at the below links:
https://api.weather.gov/alerts.atom
https://api.weather.gov/alerts/active.atom

I can currently fetch data from these endpoints but I see no documentation for them at https://www.weather.gov/documentation/services-web-api. Does any exist?

**Comment by dwhitemv25** [comment, marked as answer]
What documentation are you looking for? Should be the same as the normal endpoint without the '.atom' extension (which appears to be a shortcut to request the `application/atom+xml` output instead of using the Accept header).

**Reply by JJKraw** [reply]
A few quick checks says this is correct.  Thanks.  Now someone cut and paste that into the docs. :)
